src/components/Modals/Onboarding/Onboarding.scss

```diff
@@ -245,6 +250,10 @@ input {
         margin-bottom: 2rem;
       }
     }
+
+    &--description {
+      margin-bottom: 1rem !important;
```

Comment:
`!important` is not really best practice. can you think of an alternative solution?